### PR TITLE
KEYCLOAK-5688 Externalizer for entities shared across dc

### PR DESCRIPTION
**Pull request by hmlnarik**

### CODE REVIEW

File: distribution/pom.xml [hunk 1]
```diff
@@ -31,6 +31,10 @@
     <artifactId>keycloak-distribution-parent</artifactId>
     <packaging>pom</packaging>
 
+    <properties>
+        <keycloak.provisioning.xml>server-provisioning.xml</keycloak.provisioning.xml>
+    </properties>
+
```
Comment (mposolda) [inline distribution/pom.xml]: Didn't you included this by accident?

Comment (mposolda) [reply distribution/pom.xml]: Sorry, as long as it's ok to cover both independent changes in single PR, it looks ok.

**Comment by mposolda** [comment]
@hmlnarik Thanks! 

BTV. There are a bit more objects, which are sent over the network and may need externalizers. Especially: 
- All the sublcasses of ClusterEvent in the project. Many of them are not triggered so often, so maybe externalizer is not big issue (EG. RealmRemovedEvent). Also I am not sure if the predicates and mappers from package org.keycloak.models.sessions.infinispan.stream doesn't need externalizers? Many of them is called only in local streams, but not sure if all...

Anyway, this can be done later though. I am fine with the changes in this PR

**Review by mposolda — APPROVED** [review]

**Comment by hmlnarik** [comment]
@mposolda Thanks. Yup, I've not done the `ClusterEvent`s intentionally - the amount of the instances is not that big and I've tried to address the most used ones in cross-DC setup.

I'm merging the current changes but some more might come in a separate PR for KEYCLOAK-5688 so I'm not closing it immediately (especially the predicates - I'll need to have a look)

**Pull request merged**